tests/integration/test_vulnerability_detector/test_providers/test_providers_no_os.py

```diff
@@ -57,115 +57,161 @@
     - providers
 '''
 import os
-
 import pytest
-from wazuh_testing.modules import vulnerability_detector as vd
-from wazuh_testing.tools import LOG_FILE_PATH
-from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
-from wazuh_testing.tools.monitoring import FileMonitor
-from wazuh_testing.tools.services import control_service
-# from wazuh_testing.vulnerability_detector import make_vuln_callback, VULN_DETECTOR_EXTENDED_GLOBAL_TIMEOUT
-# Disabled until refactor
 
+from datetime import date
+from wazuh_testing.db_interface import cve_db
```

Comment:
```suggestion
from datetime import date

from wazuh_testing.db_interface import cve_db
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/02e46332a8d4b6f7aacf789e3157df9457812718